packages/scene-composer/src/layouts/SceneLayout/components/TabbedPanelContainer.tsx

```diff
@@ -1,6 +1,8 @@
 import React from 'react';
 import { Tabs } from '@awsui/components-react';
 
+import { TABBED_PANEL_CONTAINER_NAME } from '../../../common/internalConstants';
```

Comment:
Nit: Consider using module aliases for shared items to reduce coupling and make imports easier to read.

Reply:
I will try it for my next change so I can have more test when adding the alias